.prettierignore

```diff
@@ -0,0 +1,5 @@
+.yarn
+packages/*/lib
+coverage
+node_modules
+API.md
```

Comment:
This file got added.

Reply:
Per John's comment, we should also ignore CHANGELOG.md here

Reply:
Added in [70db675](https://github.com/aws-amplify/amplify-cli/pull/12020/commits/70db675322f59cd6526f9e19a7e3135383fcdccb)